experiments/functional/backup_and_restore/test.yml

```diff
@@ -54,6 +67,49 @@
           include_tasks: "./backup-restore.yml"
           vars:
             action: 'BACKUP'
+            velero_backup_name: "{{ backup_name }}"
+          when: lookup('env','COMPONENT_FAILURE') == ''
+
+        - block: 
+
+           - include_tasks: "./backup-restore.yml"
+             vars:
+               action: 'SCHEDULE'
+
+           - name: Getting the backup name
```

Comment:
Need to add a check if first backup is not in completed or failed state.

Reply:
In task ` Getting the target IP`, it should be from target service not from pod.

Reply:
I think it works with both. will not make any difference.